Sources/App/Core/AppEnvironment.swift

```diff
@@ -27,7 +28,8 @@ extension AppEnvironment {
             Environment
                 .get("ROLLBAR_LOG_LEVEL")
                 .flatMap(AppError.Level.init(rawValue:)) ?? .critical },
-        shell: .live
+        shell: .live,
+        siteURL: { Environment.get("SITE_URL") ?? "http://localhost:8080" }
```

Comment:
I've added `SITE_URL` to our Gitlab variables as `http://173.255.229.82:8080` for now.

Reply:
Perfect, this is the ideal way to solve this problem. 👍